README.md

```diff
@@ -119,14 +119,15 @@ The tables below provide an overview of the machine learning properties.
 |      [FASTREAD](#fastread)      | :white_check_mark: |             ≥1R             |
 |        [Rayyan](#rayyan)        | :white_check_mark: |        ≥50 with ≥5R         |
 |  [RobotAnalyst](#robotanalyst)  | :white_check_mark: |             ≥1R             |
-|  [SWIFT-Review](#swift-review)  | :white_check_mark: |        ≥10 with ≥1R         |
+|  [SWIFT-Active Screener](#swift-activescreener)  | :white_check_mark:<sup>2</sup> |        ≥1R+≥1I         |
```

Comment:
> SWIFT-Active Screener can use a training seed to help the software to build the prioritization model more quickly. However, providing a seed is optional and the seed can only be provided before the screening process starts. Currently, Active Screener allows users to upload only relevant documents as a training seed and ideally, these documents would be a ‘representative’ sample of the types of the documents you expect to encounter in the dataset.

So then it should be `≥1R`. If no initial training seed is provided, then:
> Screening proceeds in “batches” of a fixed size, n (here n = 30). After each set of n articles has been reviewed, the model is retrained using all previously screened articles, and the remaining articles are then re-sorted according to this revised model. In this way, the predictions for relevance are updated every 30 documents to incorporate feedback from users as screening continues.

So in that case it would be `≥30 with ≥1R`.

So minimum training data depends on whether the user provides an initial training seed. It can be either `≥1R` or `≥30 with ≥1R`, maybe we should mention these both and explain this with a footnote?

Reply:
solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53